qiskit_cold_atom/spins/spins_gate_library.py

```diff
@@ -347,3 +346,26 @@ def generator(self) -> SpinOp:
 def lxly(self, gamma: float, wires: List[int], label=None):
     """Add the LxLy gate to a QuantumCircuit."""
     return self.append(LxLyGate(gamma=gamma, label=label), qargs=wires)
+
+
+class LoadSpins(Instruction):
+    """
+    LoadSpins makes it possible to define the spin length of each qudit mode.
+
+    **Circuit symbol:**
+
+    .. parsed-literal::
+
+             ┌──────┐
+        q_0: ┤ Load ├
+             └──────┘
+    """
+    def __init__(self, num_atoms:int) -> None:
+        """Initialise new load instruction."""
+        super().__init__(name="load", num_qubits=1, num_clbits=0, params=[num_atoms], label=None)
+
+@add_gate
+def load_spins(self, wire, num_atoms):
```

Comment:
I think we should invert the order here to be consistent with the other circuit instructions in Qiskit (and add type hints).
```suggestion
def load_spins(self, num_atoms: int, wire: int):
```

Reply:
Feels to me like they should all be implemented now.